docs/source/api.rst

```diff
@@ -10,6 +10,6 @@ Core
 GeoPandas integration
 ---------------------
 
-.. automodule:: pyogrio
-    :members: read_dataframe, write_dataframe
-    :noindex:
+.. automodule:: pyogrio.geopandas
```

Comment:
```suggestion
```

I believe that this line can be removed since you are listing both functions manually below.

Reply:
Ok, let's see

Reply:
So instead of https://pyogrio--130.org.readthedocs.build/en/130/api.html#module-pyogrio.geopandas, the link now becomes https://pyogrio--130.org.readthedocs.build/en/130/api.html#geopandas-integration. Is that ok?